core/domain/html_cleaner_test.py

```diff
@@ -27,6 +27,20 @@ def setUp(self):
         super(HtmlCleanerUnitTests, self).setUp()
         self.longMessage = True
 
+    def test_filter_a(self):
```

Comment:
Hi @apb7, PTAL Thanks!

Reply:
@NishealJ, these tests seem to be a bit unclear -- can you please follow the guidelines given [here](https://github.com/oppia/oppia/wiki/Writing-backend-tests#guidelines-for-writing-good-tests)? Also, please perform each assertion individually (instead of looping through a list). Thanks!